pages/docs/config/code-config.mdx

```diff
@@ -78,7 +78,7 @@ If you want to set a variable called `production_hubspot_api_key` for use only i
 
 <Alert variant="warning">
   <p className="mb-0">
-    ⚠️ <code>APP</code> (the object type) is surrounded by double underscores.
```

Comment:
This was a weird unicode character lying around.

Reply:
I think we really do want to show this emoji:

<img width="1020" alt="Screen Shot 2021-12-09 at 4 28 48 PM" src="https://user-images.githubusercontent.com/303226/145496751-d60afd98-7179-45d6-8c46-e3a0142683a3.png">